portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsUpgrade.testcase

```diff
@@ -21,6 +21,7 @@ definition {
 	@priority = "4"
 	test CanViewRemoteAppCustomElementPortletAfterUpgrade7413 {
 		property portal.acceptance = "true";
+		property test.run.environment = "EE";
 		property data.archive.type = "data-archive-remote-app";
 		property portal.version = "7.4.13";
```

Comment:
Hi @Hijky 

Could you reorder the properties in alphabetical ascending order? And also update test CanViewRemoteAppFieldsAfterUpgrade73101.

Thanks!

Reply:
ok, I will edit and re-pull it today